x/programs/runtime/util_test.go

```diff
@@ -194,6 +194,11 @@ func (t *testRuntime) AddProgram(programID ProgramID, programName string) error
 	return t.StateManager.(TestStateManager).CompileAndSetProgram(programID, programName)
 }
 
+// AddProgramInstance creates a new account with the program [programID]
+func (t *testRuntime) AddProgramInstance(programID ProgramID, account codec.Address) {
+	t.StateManager.(TestStateManager).AccountMap[account] = string(programID)
+}
```

Comment:
This looks like a footgun for creating inconsistent state. This code is only used in the private context, right? Can we just inline it?